<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportData</class>
 <widget class="QDialog" name="ExportData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>77</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>导出数据</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>导出文件名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LineEditPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonScan">
       <property name="text">
        <string>浏览</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonOk">
       <property name="text">
        <string>导出</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ButtonOk</sender>
   <signal>clicked()</signal>
   <receiver>ExportData</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>ExportData</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
